Autopilot/AttitudeManager/Inc/AttitudeDatatypes.hpp

```diff
@@ -44,5 +44,28 @@ struct Airspeed_Data_t
     float utcTime;          
 };
 
+typedef struct {
```

Comment:
I think it would be best to just do `struct STRUCT_NAME` instead of using `typedef` to keep consistency with the other class declarations.